ts/Extensions/Annotations/Popup.ts

```diff
@@ -795,162 +875,381 @@ H.Popup.prototype = {
+                buttonParentDiv as HTMLDOMElement,
                 lang.removeButton || 'remove',
                 'remove',
-                callback,
-                buttonParentDiv as any
+                buttonParentDiv as HTMLDOMElement,
+                callback
             );
         },
+
+        /**
+         * Filter object of series which are not indicators.
+         * If the filter string exists, check against it.
+         *
+         * @private
+         *
+         * @param {Highcharts.FilteredSeries} series
+         *        All series are available in the plotOptions.
+         *
+         * @param {string|undefined} filter
+         *        Applied filter string from the input.
+         *        For the first iteration, it's an empty string.
+         *
+         * @return {void}
+         */
+        filterSeries: function (
+            this: Highcharts.Popup,
+            series: Series,
+            filter?: string
+        ): [Highcharts.FilteredSeries] {
+            const popup = this,
+                indicators = popup.indicators;
+            let filteredSeriesArray: [Highcharts.FilteredSeries] = [] as any,
+                filteredSeries: Highcharts.FilteredSeries;
+
+
+            objectEach(series, function (
+                series: Series,
+                value: string
+            ): void {
+                const seriesOptions = series.options;
+                // Allow only indicators.
+                if (
+                    (series as any).params || seriesOptions &&
+                    (seriesOptions as any).params
+                ) {
+                    const { indicatorFullName, indicatorType } =
+                        indicators.getNameType(series, value);
+
+                    if (filter) {
+                        const slicedValue = indicatorFullName.slice(0, filter.length);
+                        if (slicedValue.toLocaleLowerCase() === filter.toLocaleLowerCase()) {
+                            filteredSeries = {
+                                indicatorFullName,
+                                indicatorType,
+                                series
+                            };
+
+                            filteredSeriesArray.push(filteredSeries);
+                        }
+                    } else {
+                        filteredSeries = {
+                            indicatorFullName,
+                            indicatorType,
+                            series
+                        };
+
+                        filteredSeriesArray.push(filteredSeries);
+                    }
+                }
+            });
+
+            // Sort indicators alphabeticaly.
+            return filteredSeriesArray.sort(function (a, b): number {
+                const seriesAName = a.indicatorFullName.toLowerCase(),
+                    seriesBName = b.indicatorFullName.toLowerCase();
+                return (seriesAName < seriesBName) ? -1 : (seriesAName > seriesBName) ? 1 : 0;
```

Comment:
Do we need all of this? Simple `(a, b) => a.indicatorFullName > b.indicatorFullName` won't be enough? We are comparing strings here, right?

Reply:
I'm afraid we do need this because the sorting function should return the number.
Having capital and lower case letters also makes sorting inaccurate.